### ISSUE TITLE
AttributeError: 'property' object has no attribute 'replace'

### ISSUE THREAD

**Issue by pstroe** [dbamman/latin-bert#3]
hi there,

i would like to use latin bert and by running the command from the readme:

`python3 scripts/gen_berts.py --bertPath models/latin_bert/ --tokenizerPath models/subword_tokenizer_latin/latin.subword.encoder > berts.output.txt`

```
(base) ubuntu@me:~/latin-bert-master$ python3 scripts/gen_berts.py --bertPath models/latin_bert/ --tokenizerPath models/subword_tokenizer_latin/latin.subword.encoder > berts.output.txt
Traceback (most recent call last):
  File "scripts/gen_berts.py", line 2, in <module>
    from cltk.tokenize.word import WordTokenizer
  File "/home/ubuntu/anaconda3/lib/python3.8/site-packages/cltk/tokenize/word.py", line 25, in <module>
    from cltk.tokenize.greek.sentence import GreekRegexSentenceTokenizer
  File "/home/ubuntu/anaconda3/lib/python3.8/site-packages/cltk/tokenize/greek/sentence.py", line 10, in <module>
    from cltk.tokenize.sentence import BaseSentenceTokenizer, BaseRegexSentenceTokenizer, BasePunktSentenceTokenizer
  File "/home/ubuntu/anaconda3/lib/python3.8/site-packages/cltk/tokenize/sentence.py", line 15, in <module>
    from cltk.tokenize.latin.params import LatinLanguageVars
  File "/home/ubuntu/anaconda3/lib/python3.8/site-packages/cltk/tokenize/latin/params.py", line 155, in <module>
    class LatinLanguageVars(PunktLanguageVars):
  File "/home/ubuntu/anaconda3/lib/python3.8/site-packages/cltk/tokenize/latin/params.py", line 156, in LatinLanguageVars
    _re_non_word_chars = PunktLanguageVars._re_non_word_chars.replace("'",'')
AttributeError: 'property' object has no attribute 'replace'
``` 
i assume this error comes from cltk. i have the correct version installed:
```python
Python 3.8.8 (default, Apr 13 2021, 19:58:26) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import cltk
>>> cltk.__version__
'0.1.121'
```
upgrading to the most recent version of cltk doesn't work either.

do you have an idea how i could get rid of this error?

best,

phillip

**Comment by diyclassics** [comment]
This may be because of a dependency mismatch between CLTK v0.1.121 and NLTK. Can you try downgrading to NLTK==3.5.0? (Some background in this Issue: https://github.com/cltk/cltk/issues/1090) If that works, I'll see about refactoring here to avoid this problem with future installations. Thank you and looking forward to seeing your LatinBERT applications, PJB

**Comment by pstroe** [comment]
hi patrick, thanks for your prompt response! it was very helpful. downgrading nltk to version 3.5.0 helped and solved the issue. 
just as a side remark: as a requirement you listed tensorflow version 2.3.0. i couldn't manage to install it neither with pip nor conda. i ended up using 2.6.0, and the minimal example still ran through without problems.
and installing pygame was some kind of a nightmare (sdl2 along with fft, sdl_image, sdl_mixer, etc. are all dependencies which were missing).
i'll gladly report on results using latinbert as soon as they're there ;-)

**Comment by diyclassics** [comment]
Glad it worked. I'll see about fixing the CLTK/NLTK issue in requirements or at least better documenting it.

Also, thank you for noting the other installation issues—I will do a clean install, see what I can learn about TF, pygame, etc., and then move those concerns to a new issue.

**Comment by diyclassics** [comment]
Fixed by #5
